Packs/GSuiteSecurityAlertCenter/Integrations/GSuiteSecurityAlertCenter/GSuiteSecurityAlertCenter.py

```diff
@@ -0,0 +1,719 @@
+
+    # Create HR
+    hr = prepare_hr_for_alert_feedback(list_alert_feedback_response["feedback"])
+
+    # Create entry context
+    custom_ec = gsuite_client.remove_empty_entities(list_alert_feedback_response["feedback"])
+
+    return CommandResults(
+        outputs_prefix=OUTPUT_PATHS['FEEDBACK'],
+        outputs_key_field='feedbackId',
+        outputs=custom_ec,
+        readable_output=hr,
+        raw_response=list_alert_feedback_response
+    )
+
+
+@logger
+def fetch_incidents(client, last_run: Dict, params: Dict, is_test: bool = False) -> \
+        Tuple[Optional[list], Optional[dict]]:
+    """
+    This function is called for fetching incidents.
+    This function gets all alerts, then after get latest feedback for each alert.
+    This function will execute each interval (default is 1 minute).
+
+    :param client: Client object.
+    :param last_run: A dict with a key containing the latest incident created time we got
+        from last fetch
+    :param params: arguments for fetch-incident.
+    :param is_test: True if fetch-incident is called from test-module.
+
+    :return: A tuple containing two elements:
+            next_run (``Dict[str, int]``): Contains the timestamp that will be
+                    used in ``last_run`` on the next fetch.
+            incidents (``List[dict]``): List of incidents that will be created in XSOAR.
+    """
+
+    admin_email = params.get('admin_email')
+
+    fetch_feedback = params.get('fetch_feedback', False)
+    # Validate arguments
+    params, last_fetch = validate_params_for_fetch_incidents(params, last_run)
+
+    # Fetch Alerts API call
+    client.set_authorized_http(scopes=SCOPES['ALERT'], subject=admin_email)
+    response = client.http_request(url_suffix=URL_SUFFIX['LIST_ALERTS'], method='GET', params=params)
+
+    alerts = response.get('alerts', [])
+    next_page_token = response.get('nextPageToken', '')
+
+    demisto.info(f'[GSAC ALERT]: Request URL: {BASE_URL}{URL_SUFFIX["LIST_ALERTS"]}')
+    demisto.info(f'[GSAC ALERT]: Next Token: {next_page_token}')
+
+    incidents: List[Dict[str, Any]] = []
+
+    # Prepare incidents data
+    for alert in alerts:
+        if fetch_feedback:
+            # Fetch Alert Feedback API call
+            feedback_response = client.http_request(url_suffix=URL_SUFFIX['FEEDBACK'].format(alert.get('alertId')),
+                                                    method='GET')
+            feedback_response = feedback_response.get('feedback', [])
+            if len(feedback_response) > 0:
+                # Fetch latest feedback
+                alert['feedback'] = feedback_response[0]
+        incident = {
+            'name': f'{alert.get("type")} - {alert.get("source")}',
+            'rawJSON': json.dumps(alert)
+        }
+        incidents.append(incident)
+
+    # Fetch createTime of latest alert
+    if len(alerts) > 0:
+        timestamp = alerts[-1]['createTime']
```

Comment:
While validating params for fetch incidents in `validate_params_for_fetch_incidents` method we have added `orderBy` key so that while fetching alerts from Alert centre the alerts come in sorted manner. 
![image](https://user-images.githubusercontent.com/60967033/108345675-90d13f00-7204-11eb-84b2-3cf4452651c8.png)